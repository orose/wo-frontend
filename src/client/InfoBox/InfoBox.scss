.info-box {
  border: 1px dashed blue;
  p {
    padding: 0 10px;
  }
}
