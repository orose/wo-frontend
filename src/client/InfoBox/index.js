import React, { Component } from 'react';

import ComponentHeader from '../common/ComponentHeader';

import './InfoBox.scss';

class InfoBox extends Component {
  componentDidMount() {
    const { dispatch } = this.props;
  }

  render() {
    return (
      <section className="info-box">
        <ComponentHeader text={this.props.title} />
        <p>{this.props.text}</p>
      </section>
    );
  }
}

export default InfoBox;
