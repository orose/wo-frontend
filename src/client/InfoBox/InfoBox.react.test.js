import React from 'react';
import InfoBox from './index';
import renderer from 'react-test-renderer';

test('Test that InfoBox renders as expected', () => {
  const component = renderer.create(
    <InfoBox title="the title" text="the text" />,
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
