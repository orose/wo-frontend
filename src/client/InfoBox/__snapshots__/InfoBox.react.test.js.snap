// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test that InfoBox renders as expected 1`] = `
<section
  className="info-box"
>
  <h1
    className="component-header"
  >
    the title
  </h1>
  <p>
    the text
  </p>
</section>
`;
